lib/committee/schema_validator/open_api_3.rb

```diff
@@ -80,24 +92,16 @@ def request_unpack(request)
         request.env[validator_option.path_hash_key] = coerce_path_params
 
         query_param = unpacker.unpack_query_params(request)
+        query_param.merge!(request_param) if request.get? && validator_option.allow_get_body
+        request.env[validator_option.query_hash_key] = query_param
+      end
 
+      def copy_coerced_data_to_params(request)
         request.env[validator_option.params_key] = Committee::Utils.indifferent_hash
-        request.env[validator_option.params_key].merge!(Committee::Utils.deep_copy(query_param))
         request.env[validator_option.params_key].merge!(Committee::Utils.deep_copy(request.env[validator_option.request_body_hash_key]))
+        request.env[validator_option.params_key].merge!(Committee::Utils.deep_copy(request.env[validator_option.query_hash_key]))
```

Comment:
I think it's very good to prioritize path, query, and body in that order, just like Rails.
But this is breaking changes so I add backward compatibility code for the users which want to old priority.
(This is memo. You do not need to fix it :) )